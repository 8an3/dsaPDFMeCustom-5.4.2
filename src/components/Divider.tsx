import React from 'react';

const Divider = () => (
  <div style={{ marginTop: '0.5rem', marginBottom: '0.5rem', borderBottom: '1px solid #eee' }} />
);

export default Divider;
