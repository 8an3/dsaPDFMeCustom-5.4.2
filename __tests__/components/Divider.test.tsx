/**
 * @jest-environment jsdom
 */

import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import Divider from '../../src/components/Divider';

test('renders a message', () => {
  const { container } = render(<Divider />);
  expect(container.firstChild).toMatchSnapshot();
});
