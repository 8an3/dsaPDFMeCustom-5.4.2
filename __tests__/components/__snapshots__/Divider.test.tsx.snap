// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a message 1`] = `
<div
  style="margin-top: 0.25rem; margin-bottom: 0.25rem; border-bottom: 1px solid #eee;"
/>
`;
